lib/pmdtester/pmd_report_detail.rb

```diff
@@ -0,0 +1,23 @@
+require 'json'
+
+module PmdTester
+  # This class represents all details about report of pmd
+  class PmdReportDetail
+    attr_reader :execution_time
+    attr_reader :time_stamp
+
+    def self.save(report_info_path, execution_time, time_stamp)
```

Comment:
This looks a bit asymmetric: the save method is a class method, but the load method is a instance method.
I'd make the save method a instance method, too (with just one argument: report_info_path) and use the attributes instead of arguments.